connection.go

```diff
@@ -200,7 +200,9 @@ func HGet(RConn *redigo.Conn, key string, HKey string) (interface{}, error) {
 
 func HMGet(RConn *redigo.Conn, key string, hashKeys ...string) ([]interface{}, error) {
 	args := []interface{}{key}
-	args = append(args, hashKeys)
```

Comment:
Have you tried this -  `args = append(args, hasKeys...)`

Reply:
func HMGet1(key string, hashKeys ...string) {
	args := []interface{}{key}
	for _, v := range hashKeys {
		args = append(args, v)
	}
}

func HMGet2(key string, hashKeys ...string) {
	args := make([]interface{}, 1+len(hashKeys))
	args[0] = key
	for i, v := range hashKeys {
		args[i+1] = v
	}
}

func HMGet3(key string, hashKeys ...interface{}) {
	args := []interface{}{key}
	args = append(args, hashKeys...)
}

3000000	       602 ns/op
3000000	       417 ns/op
5000000	       259 ns/op


I bench test the three method make a string and []string to a []string. The HMGet3 is best, so i suggest modifying the function parameter type to ...interface{}. 
like this
func HMGet(RConn *redigo.Conn, key string, hashKeys ...interface{})